src/components/DropDown.jsx

```diff
@@ -0,0 +1,31 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+export const DropDown = () => {
+  const [state, setState] = useState(false);
+  const showDropdown = () => {
+    setState(true);
+  };
+  const hideDropdown = () => {
+    setState(false);
+  };
+  return (
+    <StyledDrowpdown onClick={showDropdown} onMouseLeave={hideDropdown}>
+      Filter for...
+      {state ? (
+        <ul onClick={showDropdown}>
+          <li>Camera Bodys</li>
+          <li>Lenses</li>
+        </ul>
+      ) : null}
+    </StyledDrowpdown>
+  );
+};
+
```

Comment:
It is intended that you can't select one of the `list elements`or? 🤔 Then the dropdown is for now not useful? So what is the intention to include it now if not to have have a filter functionality?